### DISCUSSION TITLE
Retrieve userid in sessionobject

### DISCUSSION THREAD

**Discussion by ExordiumX** [nextauthjs/next-auth#2462] [Help]
### Question 💬

After signing in with Google, it creates a document inside the users collection containing the session object. However, I'd like to access this same document elsewhere in my code (to add data to it), but the name of the document is the userId, and the session object does not contain it. I have no idea how to retrieve this userId.

### How to reproduce ☕️

No idea

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by balazsorban44** [comment]
https://next-auth.js.org/configuration/callbacks

Check the session and jwt callbacks
